<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventCondition</class>
 <widget class="QWidget" name="EventCondition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblValue1">
       <property name="text">
        <string>Value 1:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="value1">
       <property name="minimum">
        <number>-2147483647</number>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblValue2">
       <property name="text">
        <string>Value 2:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="value2">
       <property name="minimum">
        <number>-2147483647</number>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="radNormal">
       <property name="text">
        <string>Normal</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="typeNormal"/>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="radFlags">
       <property name="text">
        <string>Flags</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QVBoxLayout" name="layoutFlags">
       <item>
        <widget class="QComboBox" name="typeFlags"/>
       </item>
       <item>
        <widget class="ActionMap" name="flagStates" native="true"/>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QRadioButton" name="radScript">
       <property name="text">
        <string>Script</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="ServerScript" name="script" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblCompareMode">
       <property name="text">
        <string>Compare Mode:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="compareMode">
       <item>
        <property name="text">
         <string>Default</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Exists</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt; (or NaN)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Between</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblNegate">
       <property name="text">
        <string>Negate:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="negate">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionMap</class>
   <extends>QWidget</extends>
   <header>ActionMap.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ServerScript</class>
   <extends>QWidget</extends>
   <header>ServerScript.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
